content/mqtt/index.textile

```diff
@@ -0,0 +1,103 @@
+---
+title: MQTT Adapter
+section: general
+index: 105
+jump_to:
+  Help with:
+    - Using MQTT#use
+    - Decoding MQTT messages#decoding
+    - When to use MQTT#use-cases
+    - Supported features#supported
+  Security:
+    - Security#security
+    - Token authentication#token-auth
+    - SSL#ssl
+  Extra information:
+    - Adapters#adapter-info
+    - MQTT#mqtt-info
+    - Tutorials#tutorials
+---
+
+The "Ably MQTT protocol adapter":https://www.ably.io/adapters is able to convert communication from "MQTT":http://mqtt.org/ to Ably's own protocol, allowing for seamless integration of any systems you may have. MQTT (MQ Telemetry Transport) is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
```

Comment:
Maybe "convert from MQTT to Ably's own ..." => "translate back and forth between MQTT and Ably's own ...", the former doesn't really sound like a two-way translator